config.yaml.defaults

```diff
@@ -718,28 +718,31 @@ gcn:
   client_id:
   client_secret:
   notice_types:
-    - FERMI_GBM_FLT_POS
-    - FERMI_GBM_GND_POS
-    - FERMI_GBM_FIN_POS
-    - FERMI_GBM_SUBTHRESH
-    - SWIFT_BAT_GRB_POS_ACK
-    - SWIFT_XRT_POSITION
-    - SWIFT_UVOT_POS
-    # - LVC_PRELIMINARY
-    # - LVC_INITIAL
-    # - LVC_UPDATE
-    # - LVC_RETRACTION
-    # - AMON_ICECUBE_COINC
-    # - AMON_ICECUBE_HESE
-    - ICECUBE_ASTROTRACK_GOLD
-    - ICECUBE_ASTROTRACK_BRONZE
-    - MAXI_UNKNOWN
-  json_notice_types:
-    - einstein_probe.wxt.alert
-    - swift.bat.guano
-    - icecube.lvk_nu_track_search
+    voevent: []
+      # - gcn.notices.svom.voevent.grm
```

Comment:
Should we document this more? Also do we want everything commented?

Reply:
I feel like unless folks want to do MMA stuff, this isn't useful. Also you need to have gcn credentials setup so when someone does that they will pick the notices they want to listen too. I don't expect people to want to use our defaults.

Reply:
But yes let's add a comment pointing to the page where these can be found.

Reply:
added @mcoughlin